lib/views/desktop/receive/widgets/DTReceiveProgress.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:dart_wormhole_gui/constants/app_constants.dart';
+import 'package:dart_wormhole_gui/views/widgets/Heading.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import '../../../../widgets/CustomLinearProgressIndicator.dart';
+import '../../widgets/DTButton.dart';
+import '../../widgets/DTFileInfo.dart';
+
+class DTReceiveProgress extends StatefulWidget {
+  final int fileSize;
+  final String fileName;
+  final double percentage;
+  final String remainingTimeString;
+
+  DTReceiveProgress(
+      this.fileSize, this.fileName, this.percentage, this.remainingTimeString);
+
+  @override
+  State<DTReceiveProgress> createState() => _ReceiveProgressState();
+}
+
+class _ReceiveProgressState extends State<DTReceiveProgress> {
+  late final DateTime startingTime;
```

Comment:
Looks like these three variables aren't used.

Reply:
Yeah you are write. Will fix in in the next PR.